remove-node.yml

```diff
@@ -8,6 +8,8 @@
 - hosts: "{{ node | default('etcd:k8s_cluster:calico_rr') }}"
   gather_facts: no
   environment: "{{ proxy_disable_env }}"
+  roles:
+    - { role: kubespray-defaults }
```

Comment:
better just remove `environment: "{{ proxy_disable_env }}"` here

Reply:
is it okay to remove it?

Reply:
those 2 tasks don't care about proxy, so yes

Reply:
cool